src/aiortc/contrib/media.py

```diff
@@ -579,6 +580,7 @@ async def __run_track(self, track, context):
                         
                         if self.__recv_times_file is not None:
```

Comment:
Conscious decision to save all the latency data in a single file for the sender and the receiver that will be easy to parse rather than parsing tons of log lines.